koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -94,6 +94,10 @@ class ContainerError(koji.GenericError):
     """Raised when container creation fails"""
     faultCode = 2001
 
+class ContainerCancelled(koji.GenericError):
+    """Raised when container creation is cancelled by OSBS"""
+    faultCode = 2002
```

Comment:
I honestly don't know, I was digging through the koji code and the `faultCode` of `2001` doesn't appear to actually mean anything to koji so I just incremented since it appears to be arbitrary.